### DISCUSSION TITLE
Frequency market

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#30] [Show and tell]
many local now channel can be  tricked to play arround

Frequency market for those that are not exclusive to localnow lineup
linear-xxx.frequency.stream/dist/ -> glewedtv roku plex localnow stirr The_trots /xxx/hls/master/playlist.m3u8

#EXTINF:-1 ,Pattrn listed on localnow
https://linear-76.frequency.stream/dist/localnow/76/hls/master/playlist.m3u8
#EXTINF:-1 ,Pattrn not the same link
https://linear-76.frequency.stream/dist/plex/76/hls/master/playlist.m3u8
#EXTINF:-1 ,Pattrn not listed on glewed
https://linear-76.frequency.stream/dist/glewedtv/76/hls/master/playlist.m3u8
#EXTINF:-1 ,Pattrn not listed on stirr
https://linear-76.frequency.stream/dist/stirr/76/hls/master/playlist.m3u8
#EXTINF:-1 ,Pattrn  not the same link
https://linear-76.frequency.stream/dist/roku/76/hls/master/playlist.m3u8

another exemple
#EXTINF:-1 ,True History (720p)
https://linear-188.frequency.stream/dist/glewedtv/188/hls/master/playlist.m3u8
#EXTINF:-1 ,True History (720p)
https://linear-188.frequency.stream/dist/roku/188/hls/master/playlist.m3u8
#EXTINF:-1 ,True History (720p)
https://linear-188.frequency.stream/dist/plex/188/hls/master/playlist.m3u8
#EXTINF:-1 ,True History (720p)
https://linear-188.frequency.stream/dist/localnow/188/hls/master/playlist.m3u8
#EXTINF:-1 ,True History (720p)
https://linear-188.frequency.stream/dist/stirr/188/hls/master/playlist.m3u8
#EXTINF:-1 ,True History (720p)
https://linear-188.frequency.stream/dist/The_trots/188/hls/master/playlist.m3u8

and anyone know where to find  The_trots  lineup ?

#EXTINF:-1,CineClub
https://linear-192.frequency.stream/dist/The_trots/192/hls/master/playlist.m3u8
#EXTINF:-1,CineClub not listed on glewed
https://linear-192.frequency.stream/dist/glewedtv/192/hls/master/playlist.m3u8

and does it worth to add all those mirror link to selected market ?

**Comment by BellezaEmporium** [comment]
The only iteration of "The Trots" i've found so far is an Australian horse racing channel.
